t/page/country.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 require 'test_helper'
-require_relative '../../lib/page/country'
+require_rel '../../lib/page/country'
+# FIXME: This test should be isolated and not require these classes. Remove when fixed.
```

Comment:
FYI I'm quite a big fan of actually opening a Github Issue every time you're going to add a FIXME or TODO, and referencing it right in place. That makes it easier to schedule time to actually fix it, and track it etc.

Reply:
Nice, I like the sound of that too 👍